src/model/event-point-model.js

```diff
@@ -17,20 +35,25 @@ export default class EventPointModel extends Observable {
     this._notify(updateType, update);
   };
 
-  updateEventPoint = (updateType, update) => {
+  updateEventPoint = async (updateType, update) => {
     const index = this.#eventPoints.findIndex((eventPoint) => eventPoint.id === update.id);
 
     if (index === -1) {
       throw new Error('Can\'t update unexisting event point');
     }
 
-    this.#eventPoints = [
-      ...this.#eventPoints.slice(0, index),
-      update,
-      ...this.#eventPoints.slice(index + 1),
-    ];
-
-    this._notify(updateType, update);
+    try {
+      const response = await this.#eventPointsApiService.updateEventPoint(update);
+      const updatedEventPoint = this.#adaptToClient(response);
+      this.#eventPoints = [
+        ...this.#eventPoints.slice(0, index),
+        update,
+        ...this.#eventPoints.slice(index + 1),
+      ];
+      this._notify(updateType, updatedEventPoint);
```

Comment:
Отделяй пробелами для большей читаемости
```suggestion
      const updatedEventPoint = this.#adaptToClient(response);
      
      this.#eventPoints = [
        ...this.#eventPoints.slice(0, index),
        update,
        ...this.#eventPoints.slice(index + 1),
      ];
      
      this._notify(updateType, updatedEventPoint);
```